tools/cpp/windows_cc_configure.bzl

```diff
@@ -226,10 +226,39 @@ def find_msvc_tool(repository_ctx, vc_path, tool):
     # C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\bin\amd64
     tool_path = vc_path + "\\bin\\amd64\\" + tool
 
-  if not repository_ctx.path(tool_path).exists:
+  if check_exist and not repository_ctx.path(tool_path).exists:
```

Comment:
Here again, please extract the error handling; don't add `check_exists`.

Reply:
Done, thanks!